packages/discord.js/src/structures/Emoji.js

```diff
@@ -56,6 +56,17 @@ class Emoji extends Base {
     return this.id && this.client.rest.cdn.emoji(this.id, this.animated ? 'gif' : 'png');
   }
 
+  /**
+   * @type {?string}
+   * @param {ImageExtension} ext the extension of emoji
+   * @returns {string}
+   */
+  getUrl(ext) {
+    const extensions = ['webp', 'png', 'jpg', 'jpeg', 'gif'];
+    if (!extensions.includes(ext) || !ext) return undefined;
```

Comment:
Do we need that? The rest cdn getter throws an error if the extension is not valid.

Reply:
> Do we need that? The rest cdn getter throws an error if the extension is not valid.

are you talking about the all `getUrl` function or you talking about this
```js
    const extensions = ['webp', 'png', 'jpg', 'jpeg', 'gif'];
    if (!extensions.includes(ext) || !ext) return undefined;
    ````
    
    anyway I made this to return undefined like `url` getter , anyway if it is not required we can remove it